File: app/src/main/java/com/might/dwan/cashcalendar/ui/screens/settings/dagger/SettingsComponent.java
package com.might.dwan.cashcalendar.ui.screens.settings.dagger;

import com.might.dwan.cashcalendar.ui.screens.settings.SettingsFragment;

import dagger.Component;

@Component(modules = {SettingsModule.class})
public interface SettingsComponent {
    void inject(SettingsFragment fragment);
}

File: app/src/main/java/com/might/dwan/cashcalendar/ui/screens/settings/dagger/SettingsComponent.kt
package com.might.dwan.cashcalendar.ui.screens.settings.dagger

import com.might.dwan.cashcalendar.ui.screens.settings.SettingsFragment

import dagger.Component

@Component(modules = arrayOf(SettingsModule::class))
interface SettingsComponent {
    fun inject(fragment: SettingsFragment)
}
